packages/mds-compliance/api.ts

```diff
@@ -159,13 +159,20 @@ function api(app: express.Express): express.Express {
   })
 
   app.get(pathsFor('/count/:rule_id'), async (req: ComplianceApiRequest, res: ComplianceApiResponse) => {
+    // /*
+    const { timestamp: query_end_date } = {
+      ...parseRequest(req, { parser: Number }).query('timestamp')
+    }
+    //* /
     if (!AllowedProviderIDs.includes(res.locals.provider_id)) {
       return res.status(401).send({ result: 'unauthorized access' })
     }
 
+    const timestamp = query_end_date || now()
```

Comment:
Totally unnecessary to do this, just don't rename to `query_end_date` when destructuring on line 163 and instead supply a default value of `now()`